cola/ops/operator_base.py

```diff
@@ -54,7 +55,7 @@ def is_xnp_array(obj, xnp):
 class AutoRegisteringPyTree(type):
     def __init__(cls, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        cls._dynamic = cls._dynamic.copy()
+        # cls._dynamic = cls._dynamic.copy()
```

Comment:
what problem are you trying to fix with this change?
we need dynamic to make it's own copy otherwise all LinearOperators will modify the same dictionary

Reply:
This is creating a problem when running the "psd_prod" from the GPU tests. I don't understand this code but this specific line was making `Transpose` not update the device of the operator that it acted on